### ISSUE TITLE
ajouter un exemple d'utilisation dans la documentation

### ISSUE THREAD

**Issue by fbaligand** [hcapitaine/french-phonetic-analyser#3]
Ce serait intéressant d'ajouter un exemple d'utilisation dans la documentation (README.md) :
- un exemple d'index utilisant l'analyzer
- un exemple d'index utilisant le filtre
- un exemple d'index utilisant le filtre avec l'option 'replace'

**Comment by zozoit** [comment]
Je me permet de relancer ce commentaire, car il n'est spécifié nulle part comment utiliser ce token filter.

**Comment by hcapitaine** [comment]
Désolé je n'ai pas eu le temps. Par contre je suis intéressé de savoir ce
dont les utilisateurs ont besoin.

J'ai un module test qui le met en oeuvre avec une conf test mais qui est la
même en prod.

Vous désirez quoi? Un fichier de settings / mapping? Comment l'installer?
Je suis vraiment preneur de besoins?




Le jeu. 19 sept. 2019 à 10:37, Vincent Bosc <notifications@github.com> a
écrit :

**Comment by zozoit** [comment]
Alors, principalement comment appeler ce token filter.

SI on veut créer un analyzer phonétique comme metaphone avec le plugin phonétique d'ES pour la 6.5, on regarde la doc : https://www.elastic.co/guide/en/elasticsearch/plugins/6.5/analysis-phonetic-token-filter.html

Il faudrait juste l'équivalent de cette doc pour votre plugin. Je ne sais pas comment on peut le mettre dans les settings, l'utiliser dans un analyzer (quel type renseigner pour créer le bon tokenfilter), et puis pourquoi pas rajouter également un exemple d'utilisation via l'api _analyze

**Comment by fbaligand** [comment]
Ces exemples d’utilisation peuvent être des simples bout de json dans le readme.md. Pas besoin de plus forcément. 
Ce qui m’intéresse, c’est ce que j’ai mis dans la description de l’issue. 

Fabien

**Comment by zozoit** [comment]
Oui, il n'y a pas forcément besoin de grand chose. Juste quelques morceaux de json.
